serverjs/util.js

```diff
@@ -189,32 +190,33 @@ var exports = {
   newCard,
   addCardToCube,
   getCardImageURL,
-  arraysEqual: function(a, b) {
+  arraysEqual(a, b) {
     if (a === b) return true;
     if (a == null || b == null) return false;
-    if (a.length != b.length) return false;
+    if (a.length !== b.length) return false;
 
-    for (var i = 0; i < a.length; ++i) {
+    for (let i = 0; i < a.length; ++i) {
       if (a[i] !== b[i]) return false;
     }
     return true;
   },
-  CSVtoArray: function(text) {
-    let ret = [''],
-      i = 0,
-      p = '',
-      s = true;
-    for (let l in text) {
-      l = text[l];
-      if ('"' === l) {
+  CSVtoArray(text) {
+    const ret = [''];
+    let i = 0;
+    let p = '';
+    let s = true;
+    for (let l of text) {
```

Comment:
I'm never of fan of naming anything `l`, is there something appropriate and less visually confusing? `c` is ok, `char` probably more obvious.

Reply:
yea, i really have no idea what these variables are since I did not write the original function. I can certainly change `l` to `character`, but I do not really know what `p` ore `s` are suppose to be.